universal-login-relayer/test/helpers/message.ts

```diff
@@ -1,10 +1,15 @@
 import {transferMessage} from '../fixtures/basicWalletContract';
 import {calculateMessageSignature} from '@universal-login/contracts';
 
-export const getMessageWith = async (from: string, privateKey : string) => {
+interface CreateMessage {
+  from: string;
+  privateKey: string;
+}
+
+export const createMessage = async ({from, privateKey}: CreateMessage) => {
```

Comment:
```
export const createMessage = async (overrride: Partial<Message>) => {
  const message = { ...transferMessage, signature: '0x', ...overrride};
```

Reply:
Move to commons?

Reply:
Can you remove duplication of `calculateMessageSignature` in favor of `createSignedMessage`?